app/models/image.rb

```diff
@@ -19,6 +22,19 @@ class Image < ActiveRecord::Base
   scoped_search :relation => :operatingsystem, :on => :id, :rename => "operatingsystem", :complete_enabled => false, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
   scoped_search :on => :user_data, :complete_value => {:true => true, :false => false}
 
+  def password_changed_changed?
```

Comment:
We have something similar in User's model. Does it make sense to extract this to a concern to not repeat ourselves?